src/Network/Http/Response.php

```diff
@@ -388,7 +388,7 @@ protected function _getJson()
             return $this->_json;
         }
         $data = json_decode($this->_body, true);
-        if ($data) {
+        if (is_array($data)) {
```

Comment:
This will create problems since you can even encode strings into json format, hence the decode value could be string too.